src/main/java/com/neprozorro/rest/mockdata/LotInfoMockDataGenerator.java

```diff
@@ -1,6 +1,8 @@
 package com.neprozorro.rest.mockdata;
 
+import com.neprozorro.model.LotInfo;
```

Comment:
added import but can't see it's usage, check please if all you import to classes is used

Reply:
I don't see any change or opinions here